gravatar/build.gradle.kts

```diff
@@ -13,14 +13,20 @@ plugins {
     alias(libs.plugins.openapi.generator)
 }
 
+val sdkVersion = providers.exec {
+    commandLine("git", "describe", "--tags", "--abbrev=0")
+}.standardOutput.asText.get().trim()
```

Comment:
This part looks good to me, aligned with Gradle docs on configuration cache compatibility. ([docs](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:requirements:external_processes)) 👍 

This `git` operation will run on every build invocation to check if cache is up-to-date, but this is very fast operation, so I don't expect any issues.

Reply:
Thank you!